Lekce-8-HW/Vehicle.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+namespace Lekce_8_HW
+{
+    internal abstract class Vehicle
+    {
+        public string Brand;
+        public string GetBrand()
+        {
+            return Brand;
+        }
+        public void SetBrand(string brand)
+        {
+            Brand = brand;
+        }
+        public int Mileage;
+        public int GetmileAge()
+        {
+            return Mileage;
+        }
+
+        public void SetmileAge(int mileAge)
+        {
+            Mileage = mileAge;
+        }
+        public abstract void UkazInfo();
+        public virtual void NastartujMotor()
```

Comment:
Nevidim child classu, co by rozsirovala tuhle virtualni metodu. Vidim jen tu, co chovani prepisuje a tu, co chovani nemeni pouziva puvodni. Strhl jsem za to 0.5 bodu, protoze to je cast zadani